docs/src/dev-guide/contributing-linting.md

```
@@ -0,0 +1,28 @@
+
+```shell
+npm run lint:check
+```
+
+To run all linting checks AND automatically fix any fixable issues:
+
+```shell
+npm run lint:fix
+```
+
+[feature-req]: https://github.com/y-scope/clp/issues/new?assignees=&labels=enhancement&projects=&template=feature-request.yml
```

Markdownlint (0.35.0)

Unused link or image reference definition: "feature-req"
Link and image reference definitions should be needed

(MD053, link-image-reference-definitions)